.fossa.yml

```
@@ -0,0 +1,9 @@
+# Check out Fossa at: https://fossa.com/
+
+version: 3
+
+server: https://app.fossa.com
+
+paths:
+  exclude:
+    - ./testdata
```

This should include known generated directories in case they end up containing a manifest file. Also, include a trailing `/` to indicate these are directories.

```suggestion
    - ./.bin/
    - ./.tmp/
    - ./testdata/
```